### ISSUE TITLE
"Blocking" of the user/group shall prevent access to the Cloud Pipeline platform via the CLI until JWT token expiration

### ISSUE THREAD

**Issue by NShaforostov** [epam/cloud-pipeline#766]
**Background**
At the moment, via the opportunities implemented at #623 access to the Cloud Pipeline via the GUI can be prevented manually (by the admin) or automatically. But via the CLI, the "blocked" user will have access to the Platform until the JWT token is valid.

For example:
1. Valid user get the access token and use the Cloud Pipeline via the CLI
2. Admin blocks that user via the GUI
3. The user will not have access to the Cloud Pipeline via the GUI, but still, have access via the CLI. Default token duration - 1 month. So, the "blocked" user could use the Platform for a long time regardless of the "blocking".

**Approach**
If the user has the "blocked" flag, he shall not have any access to the Cloud Pipeline Platform by all the means (GUI/CLI/API).

**Comment by mzueva** [comment]
@sidoruka This will require changing the approach to JWT handling: currently we fully trust JWT content (including granted roles and groups). In order to support blocked status, we'll need to reload user from DB and revalidate it.

**Comment by sidoruka** [comment]
Yup, this issue is for "not to forget the implementation in the future"
